python-clusters/create-gke-cluster/cluster.json

```diff
@@ -77,6 +77,13 @@
             "mandatory": false,
             "visibilityCondition": "model.isVpcNative"
         },
+        {
+            "name": "nodepoolTags",
+            "type": "STRINGS",
+            "label": "Network tags",
+            "description": "Network tags to each node",
+            "defaultValue": ["dss-gke-cluster"]
+        },
```

Comment:
From gcp point of view, this network tags seems to be setted at the node pool level (and then inherited by all the nodes)
Why put it in the "network" section then ? Why not in the node pool section / presets ?